signer/signer.go

```diff
@@ -137,7 +150,10 @@ func validateSignerInfo(info *SignerInfo) error {
 	errorFunc := func(s string) error {
 		return MalformedSignatureError{msg: s}
 	}
-	if err := validate(info.Payload, info.PayloadContentType, info.SignedAttributes.SigningTime, info.SignedAttributes.Expiry, errorFunc); err != nil {
+
+	sAttr := info.SignedAttributes
+	if err := validate(info.Payload, info.PayloadContentType, sAttr.VerificationPlugin, sAttr.VerificationPluginMinVersion,
```

Comment:
Just pass the info struct rather than individual fields?

Reply:
We are reusing validate function across signing and verification call. Both have different structure(SignRequest and SignerInfo)